Assets/Scripts/Puzzle/Gate.cs

```diff
@@ -52,14 +53,17 @@ public class Gate : SignalReceiver {
 		if (panCamera && camera != null && !isShowing && allowShow && isInitialized) {
 			panningQueue.Enqueue(this);
 
-			if (panningQueue.Count == 1)
-				StartCoroutine(CoShowEvent());
+			if (panningQueue.Count == 1) {
+				VignetteHighlight highlight = Instantiate(highlightPrefab, transform.position, Quaternion.identity);
+				StartCoroutine(CoShowEvent(highlight));
+			}
 		}
 		else if (animator.isInitialized)
 			animator.SetBool(isOpenHash, IsActivated);
 	}
 
-	private IEnumerator CoShowEvent() {
+	private IEnumerator CoShowEvent(VignetteHighlight highlight) {
+		highlight.WorldTarget = transform;
```

Comment:
Would be nicer to do this just below the instantiation. Maybe move it in here too?

Reply:
Would be nice yes but since they are using the same highlight I left it here to skip having to keep track and check if the highlight is instantiated already.